crawler/config.toml.example

```diff
@@ -5,9 +5,9 @@ CRAWLED_FILENAME = "publiccode.yml"
 IGNORE_UNSUPPORTEDCOUNTRIES = [ "it" ]
 
 # This URL should be visible from the crawler
-ELASTIC_URL = "http://localhost:9200"
-ELASTIC_USER = "elastic"
-ELASTIC_PWD = ""
+ELASTIC_URL = "http://devita_elasticsearch:9200"
```

Comment:
Maybe in example file we could point to localhost:9200 since this project can run both on docker engine and local env. And furthermore those vars can be easily overridden by env vars.